extensions/http/src/jolie/net/HttpProtocol.java

```diff
@@ -202,6 +203,7 @@
 		private static final String CONTENT_DISPOSITION = "contentDisposition";
 		private static final String DROP_URI_PATH = "dropURIPath";
 		private static final String CACHE_CONTROL = "cacheControl";
+		private static final String STRING_RAW_CONTENT = "stringRawContent";
```

Comment:
I'm having a really hard time finding a good name for this parameter.. :-)

Raw is a bit confusing because we have the raw data type.

What about `doNotDecode` or `getStringContent`? They are all bad ... ;-) @jolie/developers ideas for a better name for this parameter?